purchase_order_fixed_price/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1 @@
+* Ángel Rivas <angel.rivas@solvos.es>
```

Comment:
When I talked about escaping characters, I was referring to:
* Keep `.md` extension (as recommended for v17 and above).
* Then, escape characters as follows:
```suggestion
* Ángel Rivas \<angel.rivas@solvos.es\>
```